src/Features/Core/Portable/LanguageServices/SymbolDisplayService/SymbolDisplayClassificationHelper.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.DocumentationComments;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.CodeAnalysis.Classification;
+
+namespace Microsoft.CodeAnalysis.LanguageServices
+{
+    internal static class SymbolDisplayClassificationHelper
+    {
+        public static List<SymbolDisplayPart> ConvertClassifications(
```

Comment:
Unless this needs to be mutable, would prefer an ImmutableArray as hte lingua-franca here.